python/sarcasm.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Script Name: sarcasm.py
+# Script Author: Fsaev
+# Script License: GPLv3
+
+SCRIPT_NAME = 'sarcasm.py'
```

Comment:
By convention the extension should not be part of the script name:

```suggestion
SCRIPT_NAME = 'sarcasm'
```